zerver/views/reactions.py

```diff
@@ -66,3 +67,60 @@ def remove_reaction_backend(request, user_profile, message_id, emoji_name):
     do_remove_reaction_legacy(user_profile, message, emoji_name)
 
     return json_success()
+
+@has_request_variables
+def add_reaction(request, user_profile, message_id,
+                 emoji_name=REQ(default=None),
+                 emoji_code=REQ(default=None),
```

Comment:
I don't think either of these can actually be `None`, right?  I removed `default=None` and all the tests still passed...